client/state/data-layer/wpcom/activity-log/rewind/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchRewindStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
+	}, action ) );
+};
+
+const fromApi = response => ( {
+	active: response.use_rewind,
+	firstBackup: response.first_backup_when,
+	error: { message: response.error },
```

Comment:
If there's an error, should we switch over to the error handler instead of the success path?